### PR TITLE
Add is_public to databucket object 

### PR DESCRIPTION
**Pull request by zschmerber**
Adding a flag for is_public to the databucket object so we can capture this info with a boolean.
image

### CODE REVIEW

File: dictionary.json [hunk 1]
```diff
@@ -2503,6 +2503,11 @@
       "description": "The event occurred on a personal device.",
       "type": "boolean_t"
     },
+    "is_public": {
+      "caption": "Public",
+      "description": "determination of the public accessibility.",
```
Comment (floydtree) [inline dictionary.json:2508]: Append "See specific usage." to the description to be consistent with our generic attribute descriptions. 
Also, nit: capitalization of determination

**Review by floydtree — CHANGES_REQUESTED** [review]

**Review by jonrau-at-queryai — APPROVED** [review]

**Review by Aniak5 — DISMISSED** [review]
LGTM

**Review by jonrau-at-queryai — APPROVED** [review]
LGTM. Based and bucketpilled.

**Review by floydtree — APPROVED** [review]

**Review by Aniak5 — APPROVED** [review]

**Comment by davemcatcisco** [comment]
Sorry I didn't get to this until now. I see it's already merged but it looks good to me. If only all PRs could be as uncontroversial as that!

**Pull request merged**